configapp/configapp.py

```diff
@@ -109,6 +155,12 @@ def data_image(filename):
     return send_from_directory(directory, filename)
 
 
+@app.route('/image/captured/<path:img>')
+def captured_image(img):
+    img = img.split('/')
+    return send_from_directory(os.path.join(img_predict_dir, img[0]), img[1])
```

Comment:
This seems to assume that there are exactly two parts of the `img` provided to the URL. From my point of view, that is quite naive and we should take some more care with regards to handling that.